CODEOWNERS

```diff
@@ -31,3 +31,5 @@ instrumentation/racecar/ @chrisholmes @open-telemetry/ruby-contrib-maintainers @
 instrumentation/rspec/ @chrisholmes @open-telemetry/ruby-contrib-maintainers @open-telemetry/ruby-contrib-approvers @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd @simi @kaylareopelle @xuan-cao-swi
 
 instrumentation/que/ @indrekj @open-telemetry/ruby-contrib-maintainers @open-telemetry/ruby-contrib-approvers @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd @simi @kaylareopelle @xuan-cao-swi
+
+instrumentation/aws_lambda/ @indrekj @open-telemetry/ruby-contrib-maintainers @open-telemetry/ruby-contrib-approvers @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd @simi @kaylareopelle @xuan-cao-swi
```

Comment:
Ok, this might not be the best advice. We're chatting about it here: https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/879#discussion_r1496308460

Reply:
```suggestion
```

Since your name is already listed in the reviewers, I don't think we need this. 

Instead I'd recommend this approach: https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/879#discussion_r1496463167